compiler/src/scala/quoted/internal/impl/QuoteContextImpl.scala

```diff
@@ -122,7 +122,7 @@ class QuoteContextImpl private (ctx: Context) extends QuoteContext, QuoteUnpickl
       end extension
 
       extension [ThisTree <: Tree](self: ThisTree):
-        def changeOwner(newOwner: Symbol): ThisTree =
+        def adaptOwner(using newOwner: Symbol): ThisTree =
           tpd.TreeOps(self).changeNonLocalOwners(newOwner).asInstanceOf[ThisTree]
```

Comment:
When a meta-programmer calls this method explicitly, it seems better to use explicit arguments.

Reply:
I can try to handle them explicitly, but this would complicate quite a few abstractions. Mainly `TreeTraverser`, `TreeMap`, `TreeAccumulator` and `ValDef.let`

Reply:
I meant that it might be better to change `using newOwner: Symbol` to `newOwner: Symbol`. The reason is about consistency: meta-programmers are already calling the method _explicitly_, it does not make sense to make its only important argument _implicit_.

Reply:
There is only one parameter that can be passed here which is the current owner. Which is itself looking for this implicit and returning it. This has the same implicitness but requires the user to write the same parameter on every use site.